tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -178,7 +181,17 @@ def compress(self) -> None:
 
         print("Weight compression...")
         start_time = time.perf_counter()
-        self.run_info.compression_memory_usage = memory_usage(self._compress, max_usage=True)
+        if self.memory_monitor:
+            max_value_per_memory_type = monitor_memory_for_callable(
```

Comment:
@nikita-savelyevv memory_monitor could not be used on CI, am I right? Do you have results of local runs? Could you check that memory results reproducible?

Reply:
@alexsu52 

> memory_monitor could not be used on CI, am I right?

Yes

> Do you have results of local runs? Could you check that memory results reproducible?

Thanks for the suggestion. I've performed a number of experiments and observe that the reported memory values differ from run to run. For some reason when used in tests, memory monitor does not provide reliable results. Possibly, running compression in a separate process would resolve this issue.

Taking into account this finding I decided to revert that addition of memory monitor option in conformance test.